src/analysis/special_functions/trigonometric.lean

```diff
@@ -1754,8 +1761,23 @@ lemma continuous_tan : continuous (λ x : {x | cos x ≠ 0}, tan x) :=
 lemma continuous_on_tan : continuous_on tan {x | cos x ≠ 0} :=
 by { rw continuous_on_iff_continuous_restrict, convert continuous_tan }
 
-end complex
+lemma range_cos_infinite : (range complex.cos).infinite :=
```

Comment:
Should we add `real.range_exp`, `complex.range_exp`, `complex.range_cos` and `complex.range_sin`?

Reply:
I've added those lemmas.